lib/SILGen/SILGenDestructor.cpp

```diff
@@ -117,6 +118,11 @@ void SILGenFunction::injectDistributedActorDestructorLifecycleCall(
   assert(cd->isDistributedActor() &&
          "only distributed actors have transport lifecycle hooks in deinit");
 
+  // FIXME(distributed): bring back the id storage and remove this hack...!
+  if (cd->lookupDirect(ctx.Id_id).size() > 0)
+    return;
+  // end of FIXME(distributed): bring back the id storage and remove this hack...!
```

Comment:
this is hack/removed since we don't have a good way to store the ID yet... I'll need to rework it and make it a class I guess until other work lands :-|